src/client.rs

```diff
@@ -245,6 +263,16 @@ impl Pusher{
     self._trigger(channel_strings, event, payload, None)
   }
 
+  pub fn _make_url(&self) -> Url {
+    let host = match self.cluster.clone() {
+        Some(ref cluster) => format!("api-{}.pusherapp.com", cluster),
+        None => self.host.to_string()
+    };
```

Comment:
Could you make it so that if `host` is set, it is used in favour of the `cluster`. This is the convention we use in our other server libraries, e.g. https://github.com/pusher/pusher-http-ruby/blob/master/lib/pusher/client.rb#L27-L33.